File: 백준/Bronze/30889. 좌석 배치도/좌석 배치도.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int N = Integer.parseInt(br.readLine());
        
        char[][] theater = new char[10][20];
        
        for (int i = 0; i < 10; i++) {
            for (int j = 0; j < 20; j++) {
                theater[i][j] = '.';
            }
        }
        
        for (int i = 0; i < N; i++) {
            String info = br.readLine();
            char r = info.charAt(0);
            String c = info.substring(1);
            
            int row = r - 'A';
            int col = Integer.parseInt(c) - 1;
            
            theater[row][col] = 'o';
        }
        
        for (int i = 0; i < 10; i++) {
            System.out.println(new String(theater[i]));
        }
    }
}

File: 백준/Bronze/30889. 좌석 배치도/좌석 배치도.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);

    char theater[10][20];
    for (int i = 0; i < 10; i++) {
        for (int j = 0; j < 20; j++) {
            theater[i][j] = '.';
        }
    }

    for (int i = 0; i < N; i++) {
        char row[4];
        scanf("%s", row);

        char r = row[0];
        int c = 0;

        if (row[2] != '\0') {
            c = (row[1] - '0') * 10 + (row[2] - '0');
        } else {
            c = row[1] - '0';
        }

        int rowIdx = r - 'A';
        int colIdx = c - 1;
        theater[rowIdx][colIdx] = 'o';
    }

    for (int i = 0; i < 10; i++) {
        for (int j = 0; j < 20; j++) {
            printf("%c", theater[i][j]);
        }
        printf("\n");
    }

    return 0;
}
